src/igvfd/audit/file_set.py

```diff
@@ -137,27 +157,75 @@ def audit_unexpected_seqspec(value, system):
             "audit_description": "Pod5 sequence files in a file set should not be linked to a sequence specification file.",
             "audit_category": "unexpected sequence specification",
             "audit_level": "NOT_COMPLIANT"
```

Comment:
Not apart of this ticket but this should be an ERROR instead of NOT_COMPLIANT based on the definitions we have for these levels.